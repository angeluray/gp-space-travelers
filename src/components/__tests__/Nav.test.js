import renderer from 'react-test-renderer';
import { BrowserRouter } from 'react-router-dom';
import Navigation from '../Nav/Nav';

it('Test Navbar renders correctly', () => {
  const tree = renderer
    .create(
      <BrowserRouter>
        <Navigation />
      </BrowserRouter>,
    )
    .toJSON();
  expect(tree).toMatchSnapshot();
});
