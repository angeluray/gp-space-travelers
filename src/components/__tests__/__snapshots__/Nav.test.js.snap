// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Test Navbar 1`] = `
<nav
  className="border-bottom py-3 navbar navbar-expand-lg navbar-light bg-light"
>
  <div
    className="container"
  >
    <a
      className="d-flex navbar-brand"
      href="#home"
    >
      <img
        alt="Logo"
        className="logo"
        src="world.png"
      />
      <h2
        className="ps-3"
      >
        Space Travelers' Hub
      </h2>
    </a>
    <button
      aria-controls="basic-navbar-nav"
      aria-label="Toggle navigation"
      className="navbar-toggler collapsed"
      onClick={[Function]}
      type="button"
    >
      <span
        className="navbar-toggler-icon"
      />
    </button>
    <div
      aria-expanded={null}
      className="navbar-collapse collapse"
      id="basic-navbar-nav"
    >
      <div
        className="ms-auto navbar-nav"
        onKeyDown={[Function]}
      >
        <a
          className="nav-link"
          data-rr-ui-event-key={null}
          href="#"
          onClick={[Function]}
          onKeyDown={[Function]}
          role="button"
          tabIndex={0}
        >
          <a
            aria-current="page"
            className="text-primary link active"
            href="/"
            id="mylink1"
            onClick={[Function]}
            style={
              Object {
                "color": "#0d8efd",
                "textDecoration": "underline solid #0d8efd",
              }
            }
          >
            Rockets
          </a>
        </a>
        <a
          className="nav-link"
          data-rr-ui-event-key={null}
          href="#"
          onClick={[Function]}
          onKeyDown={[Function]}
          role="button"
          tabIndex={0}
        >
          <a
            className="text-primary link"
            href="/mission"
            id="mylink2"
            onClick={[Function]}
            style={
              Object {
                "color": "#0d8dfd98",
                "textDecoration": "none",
              }
            }
          >
            Missions
          </a>
        </a>
        <a
          className="nav-link"
          data-rr-ui-event-key={null}
          href="#"
          onClick={[Function]}
          onKeyDown={[Function]}
          role="button"
          tabIndex={0}
        >
          <a
            className="text-primary link"
            href="/myProfile"
            id="mylink3"
            onClick={[Function]}
            style={
              Object {
                "color": "#0d8dfd98",
                "textDecoration": "none",
              }
            }
          >
            My Profile
          </a>
        </a>
      </div>
    </div>
  </div>
</nav>
`;

exports[`Test Navbar renders correctly 1`] = `
<nav
  className="border-bottom py-3 navbar navbar-expand-lg navbar-light bg-light"
>
  <div
    className="container"
  >
    <a
      className="d-flex navbar-brand"
      href="#home"
    >
      <img
        alt="Logo"
        className="logo"
        src="world.png"
      />
      <h2
        className="ps-3"
      >
        Space Travelers' Hub
      </h2>
    </a>
    <button
      aria-controls="basic-navbar-nav"
      aria-label="Toggle navigation"
      className="navbar-toggler collapsed"
      onClick={[Function]}
      type="button"
    >
      <span
        className="navbar-toggler-icon"
      />
    </button>
    <div
      aria-expanded={null}
      className="navbar-collapse collapse"
      id="basic-navbar-nav"
    >
      <div
        className="ms-auto navbar-nav"
        onKeyDown={[Function]}
      >
        <a
          className="nav-link"
          data-rr-ui-event-key={null}
          href="#"
          onClick={[Function]}
          onKeyDown={[Function]}
          role="button"
          tabIndex={0}
        >
          <a
            aria-current="page"
            className="text-primary link active"
            href="/"
            id="mylink1"
            onClick={[Function]}
            style={
              Object {
                "color": "#0d8efd",
                "textDecoration": "underline solid #0d8efd",
              }
            }
          >
            Rockets
          </a>
        </a>
        <a
          className="nav-link"
          data-rr-ui-event-key={null}
          href="#"
          onClick={[Function]}
          onKeyDown={[Function]}
          role="button"
          tabIndex={0}
        >
          <a
            className="text-primary link"
            href="/mission"
            id="mylink2"
            onClick={[Function]}
            style={
              Object {
                "color": "#0d8dfd98",
                "textDecoration": "none",
              }
            }
          >
            Missions
          </a>
        </a>
        <a
          className="nav-link"
          data-rr-ui-event-key={null}
          href="#"
          onClick={[Function]}
          onKeyDown={[Function]}
          role="button"
          tabIndex={0}
        >
          <a
            className="text-primary link"
            href="/myProfile"
            id="mylink3"
            onClick={[Function]}
            style={
              Object {
                "color": "#0d8dfd98",
                "textDecoration": "none",
              }
            }
          >
            My Profile
          </a>
        </a>
      </div>
    </div>
  </div>
</nav>
`;

exports[`Test RocketContainer renders correctly 1`] = `null`;
